downstream/titles/aap-operator-installation/docinfo.xml

```diff
@@ -1,4 +1,4 @@
-<title>Installing on Openshift Container Platform</title>
+<title>Operating {PlatformNameShort}</title>
```

Comment:
```suggestion
<title>Operating Ansible Automation Platform</title>
```

Reply:
You can't use attributes in docinfo: it's a separate type of file to the rest of the doc.